Content.Client/Voting/VotingSystem.cs

```diff
@@ -8,8 +7,6 @@ public sealed class VotingSystem : EntitySystem
 
     public event Action<VotePlayerListResponseEvent>? VotePlayerListResponse; //Provides a list of players elligble for vote actions
 
-    [Dependency] private readonly GhostSystem _ghostSystem = default!;
-
```

Comment:
Wouldn't you be better off checking the cvar on client then?

Reply:
All of votekick cvars are server only and the event checks for eligiblity on the server already, so I don't think there's much point in also doing it on the client now